### ISSUE TITLE
Connecturl not working

### ISSUE THREAD

**Issue by ThomasMarangoni** [altmp/altv-issues#828]
If you want to connect to a server via a website (eg [https://altv.mp/#/servers](https://altv.mp/#/servers)) you will be ask where alt:V should be installed, when selecting a location, a new installation will be installed there. 

![image](https://user-images.githubusercontent.com/30683046/106452771-a0245e80-6488-11eb-961e-7139f271a2fe.png)

**Comment by Lennnart1337** [comment]
#696 is also a similar problem

**Comment by mrgharabaghi** [comment]
This problem occurs with both C# and HTML.
```js
<a href="altv://connect/12.345.67.89:7788">OPEN</a>
```
```csharp
Process.Start("altv://connect/12.345.67.89:7788");
```

**Comment by 7Hazard** [comment]
If you're trying to use the arg on your own (without windows protocol eg. through the browser) then it is important that the current working directory is the same as where the installation resides.

**Comment by ThomasMarangoni** [comment]
**Additional information:** I installed alt:V before and it was launched.

**Comment by C0kkie** [comment]
Fixed in release/2.15 & rc/3.0-rc5
